src/api/app/models/package.rb

```diff
@@ -20,6 +20,8 @@ class Package < ApplicationRecord
   accepts_nested_attributes_for :kiwi_image
 
   belongs_to :project, inverse_of: :packages
+
+  delegate :check_access?, to: :project
```

Comment:
Can you rather change the callers? One less layer of indirection...

Reply:
Changed. I added this change in another commit.